modules/virtual_machine/variables.tf

```diff
@@ -223,4 +223,5 @@ variable "custom_data" {
   description = "The custom data to supply to the machine. This can be used as a cloud-init for Linux systems."
   type        = string
   default     = null
+  # sensitive   = true
```

Comment:
Can we remove that comment ?

Reply:
Actually we can put this property back. It was a false-positive. It was not the one causing Terratest to crash Terraform.